windows_service/datadog_checks/windows_service/windows_service.py

```diff
@@ -314,6 +317,9 @@ def check(self, instance):
                 if instance.get('windows_service_startup_type_tag', False):
                     tags.append('windows_service_startup_type:{}'.format(startup_type_string))
 
+                if instance.get('collect_display_name_as_tag', False):
+                    tags.append('display_name:{}'.format(service))
+
```

Comment:
It is interesting point here. Meaning we have not seen a service specified in config and accordingly we do not know its display name. So, adding "dummy" display name tag is a good or bad thing. My gut feeling says it is "bad" since it is potentially could be inconsistent. But on other had if you do not specify anything at all than would it affect metrics search? Perhaps we can add a default display name tag per service? IDK.

Reply:
Hey @iglendd 
I'm planning on adding something like the following:
```
display_name = "Not_Found"
(...)
if instance.get('collect_display_name_as_tag', False):
  tags.append('display_name:{}'.format(display_name))
```